django/contrib/postgres/lookups.py

```diff
@@ -58,3 +58,8 @@ def process_lhs(self, qn, connection):
 class TrigramSimilar(PostgresOperatorLookup):
     lookup_name = 'trigram_similar'
     postgres_operator = '%%'
+
+
+class TrigramWordSimilar(PostgresOperatorLookup):
+    lookup_name = 'trigram_word_similar'
+    postgres_operator = '%%>'
```

Comment:
```suggestion
    postgres_operator = '<%%'
```

I might be missing something but don't we want to test the word similarity of an input against a column? Like this:

```sql
WHERE col <% 'input'
```